### DISCUSSION TITLE
What steps to take in order to send email in Windwos Server 2012R2?

### DISCUSSION THREAD

**Discussion by Saad-Samba** [snipe/snipe-it#11704] [Q&A] [answered]
Greeting to you all, I hope you are doing alright.

I've migrated our snipe-it application from Windows 10 using WAMP server to a Windows Server 2012R2 with IIS 8.5.
Our company has an SMTP server. I have set up the "SMTP E-mail" feature on IIS Manager with correct information of the server as instructed by some tutorials :
![Capture](https://user-images.githubusercontent.com/89049334/185646252-c648de8c-8783-4f14-8832-a5336d02abb3.PNG)

I've been struggling for a week now to be able to send an email, with no success.
I'm positive that my application's configuration is right, since I can send emails from the instance using WAMP. My laravel.log has no indication of an error.

![Capture](https://user-images.githubusercontent.com/89049334/185644902-58cf1e87-8e9d-4a81-819a-ce7f6bb8e57d.PNG)

Can anyone with experience in setting the application in a similar environment provide help?

**Comment by brianhoganm** [comment]
I have Server 2019 set up with SNIPEIT and my email works just fine without having to set up a external or separate SMTP server. I am currently using smtp.gmail.com.  All you need to do is simply edit the .env config file with your smtp settings. If you already have a SMTP server - just use whatever settings, you have and plug it into your .env config file.  I didn't set up SMTP using IIS. 

Once you have everything set up correctly with your SMTP, make sure that you specify which **categories** you want  your users to receive email notifications for check-in/check-outs and requests, etc.

**Comment by Saad-Samba** [comment, marked as answer]
Finally, I solved it by taking off the **tls** out and leaving the following environment variable as follows :
`MAIL_ENCRYPTION=""`

Albeit both _Laravel.log_  and _IIS Logging_ don't display any related errors as mentioned above, I have received an indicative message in the **Network** panel of the browser's DevTools (chrome) when I click the **Send Test** button : 
`stream_socket_enable_crypto(): SSL operation failed with code 1`

Which then led me to find the solution that worked for me : 
[https://stackoverflow.com/a/58619462/16640601](url)

I don't know why it worked, and I wish to understand the reason.
